sycl/test-e2e/lit.cfg.py

```diff
@@ -177,11 +177,11 @@
 # Check for sycl-preview library
 check_preview_breaking_changes_file='preview_breaking_changes_link.cpp'
 with open(check_preview_breaking_changes_file, 'w') as fp:
-    fp.write('#include <sycl/sycl.hpp>')
-    fp.write('namespace sycl { inline namespace _V1 { namespace detail {')
-    fp.write('extern void PreviewMajorReleaseMarker();')
-    fp.write('}}}')
-    fp.write('int main() { sycl::detail::PreviewMajorReleaseMarker(); return 0; }')
+    fp.write('#include <sycl/sycl.hpp>\n')
+    fp.write('namespace sycl { inline namespace _V1 { namespace detail {\n')
+    fp.write('extern void PreviewMajorReleaseMarker();\n')
+    fp.write('}}}\n')
+    fp.write('int main() { sycl::detail::PreviewMajorReleaseMarker(); return 0; }\n')
```

Comment:
It's getting pretty hard to read at this point, so maybe this is an opportunity for cleanup?
```suggestion
    print(textwrap.dedent(
        '''
        #include <sycl/sycl.hpp>
        namespace sycl { inline namespace _V1 { namespace detail {
        extern void PreviewMajorReleaseMarker();
        }}}
        int main() { sycl::detail::PreviewMajorReleaseMarker(); return 0; }
        '''
    ), file=fp)

```

plus `import textwrap` at the top of file

Reply:
That's a good shout! I'll do this in a separate patch since this change was merged in https://github.com/intel/llvm/pull/11978.

Reply:
Suggested changes applied in https://github.com/intel/llvm/pull/11992.